src/grappler/ngraph_optimizer.cc

```diff
@@ -40,8 +40,9 @@ namespace ngraph_bridge {
 Status NgraphOptimizer::Optimize(tensorflow::grappler::Cluster* cluster,
                                  const tensorflow::grappler::GrapplerItem& item,
                                  GraphDef* output) {
-  NGRAPH_VLOG(5) << "Here at NgraphOptimizer ";
-  NGRAPH_VLOG(5) << "NgraphOptimizer : grappler item id " << item.id;
+  NGRAPH_VLOG(0) << "[NGTF-OPTIMIZER] Here at NgraphOptimizer ";
+  NGRAPH_VLOG(5) << "[NGTF-OPTIMIZER] NgraphOptimizer : grappler item id "
+                 << item.id;
```

Comment:
Minor: maybe rename `[NGTF_OPTIMIZER]` in brackets to just `NGTF_OPTIMIZER` like how we have `NGTF_SUMMARY`